pkg/config/grpc_server.go

```diff
@@ -28,7 +30,9 @@ func NewGRPCServerConfig() *GRPCServerConfig {
 
 func (s *GRPCServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.EnableGRPCServer, "enable-grpc-server", false, "Enable gRPC server")
-	fs.StringVar(&s.BindPort, "grpc-server-bindport", "8090", "gPRC server bind port")
+	fs.StringVar(&s.ServerBindPort, "grpc-server-bindport", "8090", "gPRC server bind port")
+	fs.BoolVar(&s.EnableGRPCBroker, "enable-grpc-broker", false, "Enable gRPC broker")
```

Comment:
Can we use https://github.com/openshift-online/maestro/blob/b65178176cf59907febff9ac4388341eb0e7b801/pkg/config/message_broker.go#L31 to add `grpc` instead?

Reply:
I'm still considering if this makes sense.
Currently, the message broker type is used for the Maestro server to publish resources and subscribe to resource status, while the gRPC broker is mainly for the agent to publish status and subscribe to resource spec.

Since the Maestro server and gRPC broker run together in the same process, the Maestro server doesn't use gRPC to publish resources or subscribe to resource status. Therefore, enabling the gRPC broker for the Maestro server doesn't change the message type; it merely forwards resources to gRPC broker subscribers (work agents).

So, I'm unsure if we should change the meaning of the message type to accommodate this case.